src/components/Pages/SignupForm/index.js

```diff
@@ -111,9 +152,9 @@ export const SignupForm = () => {
             </div>
           </div>
 
-          {error && <p className={style.error__text}>{error}</p>}
+          {/* {error && <p className={style.error__text}>{error}</p>} */}
```

Comment:
clean up if not needed - not sure if this is to handle firebase errors on signup